index.html

```diff
@@ -319,64 +338,155 @@ <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
+            object</a>, with <a>observer</a> set to the
+            <a data-cite="WHATWG-DOM#context-object">context object</a> and
+            <a>options list</a> set to a list containing <var>options</var> as
+            its only item.</li>
+          </ol>
+          </li>
+          <li>Otherwise (<var>observer</var>'s <a>observer type</a> is
+          <code>"single"</code>), run the following steps:
+          <ol>
+            <li>If <var>options</var>'s <a>type</a> is not contained in
+            <a>supported entry types</a>, abort these steps. The user agent
+            SHOULD notify developers when this happens, for instance via a
+            console warning.</li>
+            <li>If the <a>list of registered performance observer objects</a> of
+            <a data-cite="HTML/webappapis.html#concept-relevant-global">relevant
+            global object</a> contains a <a>registered performance observer</a>
+            <var>obs</var> whose <a>observer</a> is the <a data-cite=
+            "WHATWG-DOM#context-object">context object</a>:
+              <ol>
+                <li>If <var>obs</var>'s <a>options list</a> contains a
+                <a>PerformanceObserverInit</a> item <var>currentOptions</var>
+                whose <a>type</a> is equal to <var>options</var>'s <a>type</a>,
+                replace <var>currentOptions</var> with <var>options</var> in
+                <var>obs</var>'s <a>options list</a>.</li>
+                <li>Otherwise, append a new
+                <a>registered performance observer</a> object to the <a>list of
+                registered performance observer objects</a> of
+                <a data-cite="HTML/webappapis.html#concept-relevant-global">
+                relevant global object</a>, with <a>observer</a> set to the
+                <a data-cite="WHATWG-DOM#context-object">context object</a> and
+                <a>options list</a> set to a list containing <var>options</var>
+                as its only item.</li>
+              </ol>
+            </li>
+            <li>If <var>options</var>'s <a>buffered</a> flag is set:
+              <ol>
+                <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
+                object returned by the <a href="#filter-buffer-by-name-and-type">
+                </a> algorithm with <var>buffer</var> set to <a>relevant
+                performance entry buffer</a>, <var>name</var> set to
+                <code>null</code> and <var>type</var> set to
+                <var>options</var>'s <a>type</a>.
+                </li>
+                <li>Append <var>entries</var> to the <a data-cite=
+                "WHATWG-DOM#context-object">context object</a>'s <a>observer
+                buffer</a>.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          </li>
         </ol>
+        <p class="note">A <a>PerformanceObserver</a> object should always call
```

Comment:
fwiw, since this is non-normative text, we might want to use alternates for should, must, etc: https://tools.ietf.org/id/draft-hansen-nonkeywords-non2119-04.html#rfc.section.1

Reply:
Done and fixed the other note which also had a should